### ISSUE TITLE
[Bug]: Time won't show on card unless a date is on the card

### ISSUE THREAD

**Issue by domportera** [mgmeyers/obsidian-kanban#460]
### Describe the bug

Time won't show on a card
![image](https://user-images.githubusercontent.com/6530600/152808751-fd1ccb8f-e0f6-4476-a29e-446e0771fd24.png)

(proof time is in the card)
![image](https://user-images.githubusercontent.com/6530600/152809365-aad350f0-61b9-49d3-8890-c54637a9c5a2.png)
![image](https://user-images.githubusercontent.com/6530600/152808785-ea870eeb-66a0-4a31-bb51-eda46123d7cb.png)

Unless I add a date to the card
![image](https://user-images.githubusercontent.com/6530600/152808848-a19babc5-770b-4441-950c-cf799e956c5d.png)

### Expected Behavior

Time would show regardless of if a date is there

Also, ideally, the raw date text would not show on the card

### Steps to reproduce

Create a card
Add a time to the card
observe the timeless card

To make it show a date:
create a card
add a date to the card
add a time to the card

### Operating system

Windows

**Comment by JayPalm** [comment]
I agree that it's odd that the raw date tag shows in plain text as well as the stylized date below.

**Comment by GGGomer** [comment]
This would be a great feature to have for time-planning.
